src/sources/ffmpeg.rs

```diff
@@ -0,0 +1,43 @@
+use songbird::input::{
+    error::{Error, Result},
+    Codec, Container, Input, Metadata, Reader,
+};
+use std::process::{Child, Command, Stdio};
+
+pub async fn ffmpeg_yt(mut yt: Child, metadata: Metadata, pre_args: &[&str]) -> Result<Input> {
+    let ffmpeg_args = [
+        "-f",
+        "s16le",
+        "-ac",
+        "2",
+        "-ar",
+        "48000",
+        "-acodec",
+        "pcm_f32le",
+        "-",
+    ];
```

Comment:
Find out what the flags do and comment this

Reply:
Leaving this link here: https://ffmpeg.org/ffmpeg.html#Generic-options